docs/api/apiv3/paths/membership.yml

```diff
@@ -128,58 +93,62 @@ get:
 
         *Note: A client without sufficient permissions shall not be able to test for the existence of
         a membership. That's why a 404 is returned here, even if a 403 might be more appropriate.*
-      headers: {}
-  tags:
-  - Memberships
-  description: ''
-  operationId: View_membership
-  summary: View membership
+
 patch:
+  summary: Update membership
+  operationId: update_membership
+  description: |-
+    Updates the given membership by applying the attributes provided in the body.
+
+    By providing a `notificationMessage` within the `_meta` block of the payload, the client can include a customized message to the user
+    of the updated membership. In case of a group, the message will be sent to every user belonging to the group.
+
+    By including `{ "sendNotifications": false }` within the `_meta` block of the payload, no notifications is send out at all.
+  tags:
+    - Memberships
   parameters:
-  - description: Membership id
-    example: '1'
-    in: path
-    name: id
-    required: true
-    schema:
-      type: integer
+    - name: id
+      description: Membership id
+      in: path
+      required: true
+      example: 1
+      schema:
+        type: integer
```

Comment:
Tthe `update_membership` operation should accept the same `requestBody` parameter as the form-update-membership endpoint:

```
  requestBody:
    content:
      application/json:
        schema:
          $ref: '../components/schemas/membership_write_model.yml'
        examples:
          'no notification':
            $ref: '../components/examples/membership-create-request-no-notification.yml'
          'custom message':
            $ref: '../components/examples/membership-create-request-custom-message.yml'
```

It would be nice to create specific update example instead of reusing the `create` examples, like 'update roles' to have 2 roles to a membership.

Reply:
really nice catch! thanks